rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/license/LicenseController.java

```diff
@@ -148,6 +174,10 @@ public ResponseEntity<EntityModel<License>> createLicense(
             @RequestBody License license
     ) throws TException {
         User sw360User = restControllerHelper.getSw360UserFromAuthentication();
+        List<License> sw360Licenses = licenseService.getLicenses();
+        if(restControllerHelper.checkDuplicateLicense(sw360Licenses, license.shortname)) {
+            return new ResponseEntity("sw360 component with name" + license.shortname +"already exists.", HttpStatus.CONFLICT);
```

Comment:
change it to, return new ResponseEntity("sw360 license with name" + license.shortname +"already exists.", HttpStatus.CONFLICT);

Reply:
Fixed